lib/content/utils.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const _ = require('lodash');
+const isUUID = require('validator/lib/isUUID');
+
+
+const check = {
+
+  /**
+   * Determine if an id is in uuid format; update session for error
+   *
+   * @param  {object} config global config
+   * @param {object} req - HTTP Request
+   *
+   * @returns {boolean} false-not in uuid format / true-it is!
+   */
+  id: (config, req) => {
+    if ((typeof req.params.id !== 'string') || (!isUUID(req.params.id))) {
+      _.set(req.session, '404', {
```

Comment:
While I agree, this will cause a structural change across other files not related to the workflows PR. This works now, can we leave it and make a note? I can clean this up when I clean up users (which needs to happen to get role-level functions for workflows)